README.md

```diff
@@ -5,117 +5,25 @@
-different approach where they convert the AST directly into nodes (class instances in PHP). While this approach might 
-be a little bit faster, it introduces tight coupling between the parser and the rest of the library, which we think is 
-short-sighted. Instead we decided to take a different approach, where the parser produces the AST as an associative 
-array. This will allow developers to use the C++ parser if they want more performance.
-
-#### Abstract syntax tree
-
-We introduced a builder system, using the [Builder pattern](https://en.wikipedia.org/wiki/Builder_pattern),
-for converting the AST into nodes which allows developers to implement their own builders, without changing our code.
-
-### Type system
-
-Describes the GraphQL type system and schema definition.
-
-**Package lead: [@crisu83](https://github.com/crisu83/)**
-
-#### Schema definition
-
-Most of the existing GraphQL implementations encourages the developer to create the schema programmatically. However,
-GraphQL has an experimental feature which lets you define your schema using its Schema Definition Language (SDL). We 
-think that this is the natural way to define the schema, so we built our type system around this idea. This approach 
-will also allow developers to define the schema in formats native to different ecosystems, such as PHP array, YAML and 
-even XML.
-
-### Validation
-
-**Package lead: [@crisu83](https://github.com/crisu83/)**
-
-#### Schema validation
-
-Our schema validation is based on the reference implementation, but we decided to implement it using rules. We introduced three rules, one for validating root types (query, mutation and subscription), one for validating directives and one for validating all the types.
-
-#### Query validation
-
-Even though some GraphQL implementations come without any query validation we decided to include it in the first 
-version, because it is a part of the specification. Query validation is done by evaluating the query AST using a set of validation rules (27 in total). We used the [Visitor pattern](https://en.wikipedia.org/wiki/Visitor_pattern) to implement query validation, but instead of using a functional approach, like the reference implementation, we decided to go with a more traditional, object-oriented apporach. 
+## Usage
 
-In practice this means that the query AST is visited by the each rule. Every rule takes care of validating the query against a particular part of the [Validation specification](https://facebook.github.io/graphql/October2016/#sec-Validation). This also allows developers to easily implement their own validation rules.
+### Table of contents
 
-## Usage
+- [Installation](#installation)
+- [Example](#example)
+- [Creating a schema](#creating-a-schema)
+- [Execution](#execution)
+- [Scalars](#scalars)
+- [Integration](#integration)
+- [Relay support](#relay-support)
 
 ### Installation
```

Comment:
I'd move this to a separate second-level heading before Usage

Reply:
Good idea. I’ll change this.